Project/verify_dashboard.py

```
@@ -0,0 +1,146 @@
+    print("\nVerification complete! Screenshots saved as verify_*.png")
+
+if __name__ == "__main__":
+    with sync_playwright() as p:
+        # Check if server is up
+        import http.client
+        conn = http.client.HTTPConnection("localhost", 8000)
+        try:
+            conn.request("GET", "/")
+            res = conn.getresponse()
+            if res.status != 200:
+                raise Exception(f"Server returned status {res.status}")
```

Ruff (0.15.2)

[warning] Abstract `raise` to an inner function

(TRY301)